app/src/main/java/fr/free/nrw/commons/Urls.java

```diff
@@ -0,0 +1,14 @@
+package fr.free.nrw.commons;
+
+class Urls {
+    static final String NEW_ISSUE_URL = "https://github.com/commons-app/apps-android-commons/issues";
+    static final String GITHUB_REPO_URL = "https://github.com/commons-app/apps-android-commons";
+    static final String WEBSITE_URL = "https://commons-app.github.io";
+    static final String CREDITS_URL = "https://github.com/commons-app/apps-android-commons/blob/master/CREDITS";
+    static final String FAQ_URL = "https://github.com/commons-app/apps-android-commons/wiki/Frequently-Asked-Questions";
+    static final String PLAY_STORE_URL = "https://play.google.com/store/apps/details?id=fr.free.nrw.commons";
+    static final String TRANSLATE_WIKI_URL = "https://translatewiki.net/w/i.php?title=Special:Translate&group=commons-android-public static final Strings&filter=%21translated&action=translate&language=";
```

Comment:
This string seems problematic, can you verify @madhurgupta10 ?

Reply:
@neslihanturan Fixed. Sorry for the Typo, I used "find and replace" from the android studio and did not notice, this also got changed.

Reply:
Find and replace is quite risky:)